csi/controller_server.go

```diff
@@ -440,6 +443,8 @@ func (cs *ControllerServer) ControllerPublishVolume(ctx context.Context, req *cs
 		return nil, status.Errorf(codes.Aborted, "volume %s is not ready for workloads", volumeID)
 	}
 
+	// TODO: make sure that volume is not ready to be attached if the clone hasn't finished
```

Comment:
This is resolved in resolved in `toVolumeResource`, isn't it?

Reply:
You are right. I forgot to remove this comment. Thank you. Fixed